packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -78,25 +78,28 @@ _The key elements of the component's public API surface:_
 
 -   Component Name: `nimble-wafer-map`
 -   Props/Attrs:
-    -   `die[]` - this represents the input data, which fills the wafer map with content.\
+    -   `dice[]` - this represents the input data, which fills the wafer map with content.\
```

Comment:
As I noted elsewhere in the PR, I believe "dies" is the correct term for this (see [here](https://www.merriam-webster.com/dictionary/die#dictionary-entry-2)).

Reply:
Updated